advantages.md

```diff
@@ -10,16 +10,20 @@ The advantages/disadvantages with same numbers belong together.
   1. Precomputing waves that will never be used (e.g. for unused frequencies)
 2. Code gets a bit more complicated (more communication for wavetable pointers, wavetable parameters, pasting etc. between RT and non-RT threads)
 3. Frequencies may not match exactly the played frequency (though probably without hearable difference for Nyquist-Shannon. It works in PAD synth, too)
-4. Some parameters can not be automated in realtime anymore (previously, you could change the base function for every new ADnote in realtime)
-5. -
+4. Additional latency to some parameter automation, though automations are still
+   possible. e.g. changing the waveform base function may not be reflected on a
+   note immediately after the change, but notes a short period of time after the
+   change should have the updated waveform shape
+5. Introducing random elements is more challenging
 
 ## Advantages:
 
 1. Lower computation times in RT thread:
   1. If no random (default settings), waves can be re-used
 2. The wavetable implementation makes it easier to implement wavetable modulation (e.g. by modulating the basewave parameter)
   1. OscilGen code can be non RT
-3. -
-4. -
-5. Faster reaction time: FFT/IFFT are done way before noteOn
+3. Faster reaction time: FFT/IFFT are done way before noteOn
   1. Especially, playing multiple notes at once is no problem anymore (imagine DAWs and the begin of a bar)
+4. Avoids non-realtime safe data preparation within AdNote::noteOn
+5. Allows for the possibility of selecting multiple anti-aliased wavetables with
```

Comment:
add on.

Consider note A with a single sawtooth oscillator. The note is created at frequency B, but with modulation it goes from [C..D] Hz. At the extreme positions C/D can either alias or have missing harmonics. If the transitions are slow then you could hop from wavetable to wavetable basically adding and removing harmonics as things evolve.

Not saying it's necessarily a good impl, but a possibility enabled by the new system since the whole recompute isn't needed and the others are sitting there for free (just the interpolation between tables is non-free).